index.html

```diff
@@ -282,21 +306,38 @@ <h5 class="lang-p">terminal</h5>
     </section>
     <section class="footer">
       <h2 class="footer-text">
-        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
       </h2>
       <form action="" class="form">
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the contact section(form) is propelry aligned to the center  just as it is in the [figma template 2] (https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-06-09 12-53-35](https://user-images.githubusercontent.com/58692590/172845916-03a3302a-46f4-4eb5-afdf-ec8823cda916.png) | ![Screenshot from 2022-06-09 12-53-28](https://user-images.githubusercontent.com/58692590/172846063-49ef426e-ebdd-4ce5-8734-ff34e9eace96.png)